contracts/bancor-portal/BancorPortal.sol

```diff
@@ -256,7 +260,7 @@ contract BancorPortal is IBancorPortal, ReentrancyGuardUpgradeable, Utils, Upgra
         address provider
     ) private returns (MigrationResult memory) {
         // get Uniswap's pair
-        address pairAddress = factory.getPair(address(token0), address(token1));
+        address pairAddress = factory.getPair(address(_nativeToWeth(token0)), address(_nativeToWeth(token1)));
```

Comment:
we shouldn't call `_nativeToWeth` if it's not the native token - you could add a check at the beginning of the method and if it's indeed the native token, translate it to WETH right there.